pom.xml

```diff
@@ -34,39 +35,74 @@
-      <artifactId>github-api</artifactId>
-      <version>1.67</version>
-    </dependency>
-    <dependency>
-      <groupId>org.jenkins-ci.plugins</groupId>
-      <artifactId>git</artifactId>
-      <version>2.0</version>
-    </dependency>
-    <dependency>
-      <groupId>org.jmock</groupId>
-      <artifactId>jmock-junit4</artifactId>
-      <version>2.5.1</version>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.eclipse.jgit</groupId>
-      <artifactId>org.eclipse.jgit</artifactId>
-      <version>0.12.1</version>
-    </dependency>
-    <dependency>
-      <groupId>org.jenkins-ci.plugins</groupId>
-      <artifactId>multiple-scms</artifactId>
-      <version>0.2</version>
-      <optional>true</optional>
-    </dependency>
-    <dependency><!-- exists in the core -->
-      <groupId>org.jenkins-ci.modules</groupId>
-      <artifactId>instance-identity</artifactId>
-      <version>1.3</version>
-      <scope>provided</scope>
-    </dependency>
+      
+      <dependency>
+          <groupId>org.slf4j</groupId>
+          <artifactId>slf4j-jdk14</artifactId>
+          <version>1.7.7</version>
+      </dependency>
+
+      <dependency>
+          <groupId>org.jenkins-ci.plugins</groupId>
+          <artifactId>github-api</artifactId>
+          <version>1.67</version>
+      </dependency>
+
+      <dependency>
+          <groupId>org.jenkins-ci.plugins</groupId>
+          <artifactId>git</artifactId>
+          <version>2.0</version>
+      </dependency>
+
+      <dependency>
+          <groupId>org.eclipse.jgit</groupId>
+          <artifactId>org.eclipse.jgit</artifactId>
+          <version>0.12.1</version>
+      </dependency>
+
+      <dependency>
+          <groupId>org.jenkins-ci.plugins</groupId>
+          <artifactId>multiple-scms</artifactId>
+          <version>0.2</version>
+          <optional>true</optional>
+      </dependency>
+
+      <dependency><!-- exists in the core -->
+          <groupId>org.jenkins-ci.modules</groupId>
+          <artifactId>instance-identity</artifactId>
+          <version>1.3</version>
+          <scope>provided</scope>
+      </dependency>
+
+      <!--TEST DEPS-->
+      <dependency>
+          <groupId>org.hamcrest</groupId>
+          <artifactId>hamcrest-all</artifactId>
+          <version>1.3</version>
+          <scope>test</scope>
+      </dependency>
+      
+      <dependency>
+          <groupId>junit</groupId>
+          <artifactId>junit</artifactId>
+          <version>4.12</version>
+          <scope>test</scope>
+      </dependency>
+      
+      <dependency>
+          <groupId>org.jmock</groupId>
+          <artifactId>jmock-junit4</artifactId>
+          <version>2.5.1</version>
+          <scope>test</scope>
+      </dependency>
+
+      <dependency>
+          <groupId>org.mockito</groupId>
+          <artifactId>mockito-core</artifactId>
+          <version>1.10.19</version>
+          <scope>test</scope>
+      </dependency>
+
```

Comment:
@jglick, I know about small diffs, thx! But this change alreay discussed with @KostyaSha